docs/software/mqtt/index.mdx

```diff
@@ -90,34 +116,6 @@ Check out [MQTT Settings](/docs/settings/moduleconfig/mqtt) for full information
 `uplink_enabled` will tell the device to publish mesh packets to MQTT.
 `downlink_enabled` will tell the device to subscribe to MQTT, and forward any packets from there onto the mesh.
 
-### Topics
-
-The "mesh/crypt/CHANNELID/NODEID/PORTID" [topic](https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices) will be used for messages sent from/to a mesh.
-
-Gateway nodes will forward any MeshPacket from a local mesh channel with uplink_enabled. The packet (encapsulated in a ServiceEnvelope) will remain encrypted with the key for the specified channel.
```

Comment:
Having just read through the legacy documentation, I think the references to "ServiceEnvelope" and a passing reference to "MeshPacket" were helpful. It lead me to a basic "where do I start" for interpreting the protobuf documentation.

Reply:
Good points. I added it again in: https://github.com/meshtastic/meshtastic/pull/746/commits/ee40ecec4ddbaa3a125be2bf1b77fabd720c3c45